notebooks/3.jump-analysis/nbconverted/3.jump_analysis.py

```diff
@@ -361,23 +365,65 @@
     lambda injury_code: injury_decoder[str(injury_code)]
 )
 
-# next only select cytoskeletal probability scores
-cytoskeletal_proba_scores = all_proba_scores[col_to_sel + ["Cytoskeletal"]]
-cytoskeletal_proba_scores.rename(columns={"Cytoskeletal": "Cytoskeletal Proba"})
 
+# We will now save the ground truth predictions, which include probability scores for each injury type and model type, as well as the predicted injury. These results will be stored in the `./results/3.jump_analysis` directory.
 
 # In[14]:
 
 
-# Saving only cyr
+# update columns by replacing the column index (which are the injury codes) to the injury type
+gt_y_proba_df.columns = [
+    injury_decoder[str(injury_code)]
+    for injury_code in gt_y_proba_df.columns.tolist()[0:15]
+] + col_to_sel
+shuffled_gt_y_proba_df.columns = [
+    injury_decoder[str(injury_code)]
+    for injury_code in shuffled_gt_y_proba_df.columns.tolist()[0:15]
+] + col_to_sel
```

Comment:
There might be a way to do this without using list comprehension, without the pandas api